lib/plugins/aws/provider.js

```diff
@@ -1262,6 +1262,10 @@ class AwsProvider {
             },
             kmsKeyArn: { $ref: '#/definitions/awsKmsArn' },
             layers: { $ref: '#/definitions/awsLambdaLayers' },
+            logRetentionInDays: {
```

Comment:
Maybe it could become a definition and be used both in `provider` and `function`? It would allow us to avoid potential bugs if that definition has to be updated at some point